### ISSUE TITLE
publish to nexus repo with rest api

### ISSUE THREAD

**Issue by j143** [j143/systemds#96]
https://help.sonatype.com/repomanager3/rest-and-integration-api

https://support.sonatype.com/hc/en-us/articles/213465868-Uploading-to-a-Staging-Repository-via-REST-API

```sh
curl -v -u admin:admin123 -H "Content-type: application/x-rpm" --upload-file some.rpm http://localhost:8081/nexus/service/local/staging/deployByRepositoryId/abc-1000/com/mycompany/artifact/1.0/artifact-1.0.rpm
```

**Comment by j143** [comment]
![image](https://user-images.githubusercontent.com/53068787/119823230-30901e80-bf12-11eb-9279-6bc705360d37.png)

**Comment by j143** [comment]
Param | value
-- | --
Profile Name | Test
Profile Selection Strategy | Explicit or Implicit - selected by ID or Nexus will calculate match
Searchable Repositories | ✅
Staging Mode | Deploy and UI Upload
Template | Maven2 (hosted, release)
Repository Target | All (Maven2)
Release Repository | Releases
IQ Server Application | (none)
Content Type | maven2

**Comment by j143** [comment]
![image](https://user-images.githubusercontent.com/53068787/119824971-19523080-bf14-11eb-8daf-37cb127bca5b.png)

**Comment by j143** [comment]
fixes #49
